packages/sirius-web/frontend/sirius-web/src/nodes/EllipseNode.types.ts

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLNodeStyle, NodeData } from '@eclipse-sirius/sirius-components-diagrams-reactflow';
+
+export interface EllipseNodeData extends NodeData {
+  style: React.CSSProperties;
```

Comment:
Note that in https://github.com/eclipse-sirius/sirius-web/pull/2507 the `style` attribute (present in all `NodeData` subtypes) is moved up directly in `NodeData` so this will not be needed anymore.